src/huggingface_hub/utils/tqdm.py

```diff
@@ -53,71 +53,141 @@
     for _ in tqdm(range(5)):
        do_something()
     ```
+
+Hierarchical Control Example:
+    ```python
+    # Disable progress bars for a specific group
+    disable_progress_bars("peft.foo")
+
+    # Check state of different groups
+    print(are_progress_bars_disabled("peft"))         # Outputs: False
+    print(are_progress_bars_disabled("peft.something"))  # Outputs: False
+    print(are_progress_bars_disabled("peft.foo"))       # Outputs: True
+    print(are_progress_bars_disabled("peft.foo.bar"))   # Outputs: True
```

Comment:
Usually the preferred way to document inputs and outputs is to make it look like you are running in a terminal. It renders better in the docs.

```suggestion
    ```python
    # Disable progress bars for a specific group
    >>> disable_progress_bars("peft.foo")

    # Check state of different groups
    >>> are_progress_bars_disabled("peft"))
    False
    >>> are_progress_bars_disabled("peft.something")
    False
    >>> are_progress_bars_disabled("peft.foo"))
    True
    >>> are_progress_bars_disabled("peft.foo.bar"))
    True
```

(to be adapted in the example below as well)

Reply:
Adapted 👍